pkg/autoscaler/bucket/bucket.go

```diff
@@ -29,3 +33,19 @@ func IsBucketHost(host string) bool {
 	// have host with the prefix. Maybe use regexp for improvement.
 	return strings.HasPrefix(host, prefix)
 }
+
+// AutoscalerBucketName returns the name of the Autoscaler bucket with given `ordinal`
+// and `total` bucket count.
+func AutoscalerBucketName(ordinal, total uint32) string {
+	return strings.ToLower(fmt.Sprintf("%s-%02d-of-%02d", prefix, ordinal, total))
```

Comment:
Feels weak that we don't expect to run >100 autoscaler pods :(

Reply:
The number is for bucket, not for autoscaler pods. IIRC, the #bucket is no more than 20? cc @mattmoor